packages/shared/styles/components/molecules/SfCharacteristic.scss

```diff
@@ -7,33 +7,24 @@
     display: flex;
     align-items: center;
     justify-content: center;
-    margin: var(--characteristic-margin, 0 calc(var(--spacer-big) / 2) 0 0);
+    margin: var(--characteristic-margin, 0 var(--spacer-sm) 0 0);
   }
   &__title {
-    font: var(
+    @include font(
       --characteristic-title-font,
-      font(
-        --characteristic-title,
-        500,
-        var(--font-size-regular),
-        1.6,
-        var(--body-font-family-secondary)
-      )
+      var(--font-bold),
+      var(--h5-font-size),
```

Comment:
I think here it's better to use `--font-sm`, `var(--h5-font-size)` is dedicated mainly for headings :)

Reply:
In figma was a comment /* Desktop H5 */ but sure, I'll change it :)

Reply:
@martaradziszewska @justyna-13 IMO we can use heading styles without <h> tag :) it's just styles. h5 is ok here but without h5 tag IMO

Reply:
ok, also we should remember that styles for headings have mq styles., in case we don't need any styles for desktop/mobile  :)